tests/pivots/spreadsheet_pivot/spreadsheet_pivot.test.ts

```diff
@@ -477,21 +478,6 @@ describe("Spreadsheet Pivot", () => {
     );
   });
 
-  test("Sum with a field that contains a string should work", () => {
```

Comment:
Maybe we could add a test which explicitly ensure that the sum should not work ? With `git blame` we will find this commit :)

Reply:
Yeah ... but TBH I'm not too happy with the result. The aggregation result is set to "0", which isn't great...

1) It's fine
2) we should explicitly have the pivot cells be in error if the aggregator isn't supported by the field type
3) having sum work on a text field (by having rawValue or something in the data entry probably)

wdyt ?

Reply:
I would go for 2 if it's not too big. If it's too big, go for 1

Reply:
@pro-odoo added some commits. What do you think ?

Reply:
Nice, good for me :)